Recruitment/JobSpecs/why_you_mobile_developer.md

```diff
@@ -0,0 +1,18 @@
+### What you'll do
+* Work intimately with clients to understand their organisation, aspirations and challenges.
+* Help clients identify promising solutions through participating in, and leading, workshops.
+* Build real solutions through experimentation as part of a multi-faceted team.  
+* Accelerate and improve how we help our clients through developing and contributing to components, services and methodologies.
+* Explore up-and-coming technologies and software products; seize learning and knowledge sharing opportunities; participate in professional organizations. Read; teach; learn.
+
+
+### Who fits this role
```

Comment:
Thanks bud - I like, I like! I was concerned the header didn't really fit the subsequent bullet points - James' solution nicely solves this. Tweaking now